ec2/controller.go

```diff
@@ -20,8 +21,16 @@ func CreateResourceDescription(nd *b.NamespaceDescription, instance *ec2.Instanc
 	}
 	err := rd.BuildDimensions(dd)
 	h.LogError(err)
+
+	tags := make(map[string]*string)
+	for _, t := range instance.Tags {
+		tags[*t.Key] = t.Value
+	}
+
 	rd.ID = instance.InstanceId
-	rd.Name = instance.InstanceId
```

Comment:
it should be in else block if tags["Name"] does not exists?

Reply:
Happy to do it like that if you want, but why set the name as the instance ID if the name isn't otherwise available. Given the instance ID is available in another label it seems a bit confusing to me.